_docs/setup/kubernetes/aws-nlb.md

```diff
@@ -0,0 +1,92 @@
+---
+title: AWS NLB
+overview: Instructions to use and configure ingress istio with aws NLB.
+
+order: 12
+
+layout: docs
+type: markdown
+---
+
+{% include home.html %}
+
+Instructions to use and configure ingress istio with aws NLB.
+
+## Prerequisites
+The following instructions require you have access to a Kubernetes **1.9.0 or newer** cluster
+*Warning:* This is an alpha feature and not recommended for production clusters yet.
```

Comment:
In a previous reviews related to adding  features, I was nudged away from providing indications of suitability for dev/alpha/beta/stable feature stages.  If you do want to provide such a warning, have a look at the icon here: https://github.com/istio/istio.github.io/blame/master/_docs/setup/kubernetes/quick-start-gke-dm.md#L26

In addition, the adverb yet is redundant.